fastly/computeacls/api_lookup.go

```diff
@@ -33,6 +34,10 @@ func Lookup(c *fastly.Client, i *LookupInput) (*ComputeACLEntry, error) {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
```

Comment:
In the case that no matching IP was found, the API will return a `204 No Content`. This is not an error condition, rather a lack of results.

Instead of returning an error here, consider alternatives to allow the caller to distinguish between an error (e.g. network timeout) and this case. One idea is to return `(nil, nil)`.